packages/kernel-6.12/kernel-6.12.spec

```diff
@@ -272,6 +293,18 @@ make -s\\\
 %kmake %{?_smp_mflags} M=%{_builddir}/neuron_latest
 %endif
 
+# Build EFA driver
+pushd %{_builddir}/efa_driver/build
+sed -i -e 's,$(MAKE),%{kmake},g' ../config/Makefile
+
+# Prevent polluting the parent environment by configuring CMAKE in a subshell
+(
+%{cross_cmake} ..
```

Comment:
I tried to setup the environment such that I could use `%{cross_cmake}` without the `..` but the build didn't like it.